src/Storage.cpp

```diff
@@ -46,7 +46,7 @@ typedef struct __attribute__((packed)) {
     double brewSwTimeSec;
     uint8_t freeToUse10[2];
     double brewDetectionThreshold;
-    uint8_t freeToUse11;
+    double wifisetup;
```

Comment:
```suggestion
    double wifiCredentialsSaved;
```

Reply:
I think "wifisetup" is a bit too ambiguous

Reply:
Oh and I guess double is not a good idea, use uint8_t

Reply:
good idea with the name!

Reply:
Implemented in the last commit